contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -750,4 +750,33 @@ CREATE OR REPLACE VIEW information_schema_tsql.SEQUENCES AS
 
 GRANT SELECT ON information_schema_tsql.sequences TO PUBLIC; 
 
+/*
+ * SCHEMATA view
+ */
+CREATE OR REPLACE VIEW information_schema_tsql.schemata AS
+	SELECT CAST(sys.db_name() AS sys.sysname) AS "CATALOG_NAME",
```

Comment:
not sure if calling sys.db_name repeatedly good for performance as it is being used at multiple places. Should try to see if we can have an extra join with db_name.

Reply:
I am not a big fan to add this here as well but this should not be a perfomance issue since optimizer will rewrite it as a literal in the plan.